### PR TITLE
Some rendering mappings

### PR DESCRIPTION
**Pull request by OroArmor**

### CODE REVIEW

File: mappings/net/minecraft/client/render/BuiltChunkStorage.mapping [hunk 1]
```diff
@@ -1,7 +1,15 @@
 CLASS net/minecraft/unmapped/C_rqhfsngs net/minecraft/client/render/BuiltChunkStorage
+	FIELD f_afwynqyz pastViewDistance I
```
Comment (EnnuiL) [inline mappings/net/minecraft/client/render/BuiltChunkStorage.mapping]: `past` or `last`? i should look closer in the code later

File: mappings/com/mojang/blaze3d/texture/TextureUtil.mapping [hunk 2]
```diff
@@ -39,3 +42,11 @@ CLASS com/mojang/blaze3d/platform/TextureUtil com/mojang/blaze3d/texture/Texture
 		ARG 3 mipLevels
 		ARG 4 width
 		ARG 5 height
+	METHOD writeAsPNG (Ljava/nio/file/Path;Ljava/lang/String;IIIILjava/util/function/IntUnaryOperator;)V
```
Comment (EnnuiL) [inline mappings/com/mojang/blaze3d/texture/TextureUtil.mapping:45]: Do we use `PNG` or `Png`? I feel like we should be using `Png`

**Review by EnnuiL — CHANGES_REQUESTED** [review]
Looks good! just a few things:

**Review by eristhea — APPROVED** [review]
address ennui's comments and this is great!
also: see https://github.com/QuiltMC/quilt-mappings/issues/474, you support `mipLevel` then?

**Comment by OroArmor** [comment]
Yes mipLevel is better

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/client/render/model/BakedModelManager.mapping [hunk 3]
```diff
@@ -25,20 +34,35 @@ CLASS net/minecraft/unmapped/C_wosoicga net/minecraft/client/render/model/BakedM
...
 		ARG 1 resources
 	METHOD m_nrawhhyn shouldRerender (Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_txtbiemp;)Z
 		ARG 1 from
 		ARG 2 to
+	METHOD m_oppkkipg (Lnet/minecraft/unmapped/C_eslcbfsq;Lnet/minecraft/unmapped/C_wosoicga$C_mogtdkcx;)V
+		ARG 2 reloadState
+	METHOD m_palizatf (Ljava/util/List;)Ljava/util/Map;
+		ARG 0 _list
 	METHOD m_pfipxvmw apply (Lnet/minecraft/unmapped/C_wosoicga$C_mogtdkcx;Lnet/minecraft/unmapped/C_eslcbfsq;)V
 		ARG 1 reloadSTate
```
Comment (IotaBread) [inline mappings/net/minecraft/client/render/model/BakedModelManager.mapping]: Just noticed the "s tate"

**Review by IotaBread — APPROVED** [review]

File: mappings/net/minecraft/client/render/model/BakedModelManager.mapping (hunk 3 above)
Comment (OroArmor) [reply mappings/net/minecraft/client/render/model/BakedModelManager.mapping]:
```suggestion
		ARG 1 reloadState
```

File: mappings/net/minecraft/client/render/BuiltChunkStorage.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/client/render/BuiltChunkStorage.mapping]: to note: mojang and yarn both name this `viewDistance`, no prefix

Comment (OroArmor) [reply mappings/net/minecraft/client/render/BuiltChunkStorage.mapping]: yeah ngl i kinda guessed on this one

File: mappings/com/mojang/blaze3d/texture/TextureUtil.mapping (hunk 2 above)
Comment (OroArmor) [reply mappings/com/mojang/blaze3d/texture/TextureUtil.mapping:45]: This was automapped and matches the above function. for now i say we keep it the same

**Review by EnnuiL — APPROVED** [review]

**Comment by EnnuiL** [comment]
Fast-tracking due to new Minecraft snapshot

**Pull request merged**